nuxeo-services/nuxeo-platform-query-api/src/main/java/org/nuxeo/ecm/platform/query/api/AbstractPageProvider.java

```diff
@@ -1106,4 +1106,9 @@ protected void incorporateAggregates(Map<String, Serializable> eventProps) {
         }
 
     }
+
+    @Override
+    public long getMaxResults() {
+        return 0;
+    }
```

Comment:
not really keen on saying that by default, there is no limit, but i understand that it's easier not to implement the method on all page providers...

still does not really make sense to me to have that default behaviour, especially if this max is ignored by most of implementations